src/assets/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,27 @@
+.alert {
+  display: flex;
+  align-items: flex-start;
+  width: 100%;
+  margin-bottom: rem(20px);
+  padding: rem(20px);
+
+  border-radius: rem(12px);
+
+  background-color: rgba(37, 37, 37, 0.6);
```

Comment:
How about creating a variable for this as well?

Reply:
Added in 36badc095034574ec756079e56899165c3711bfa